File: test2/app/src/main/java/com/example/test2/MainActivity.java
package com.example.test2;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;

public class MainActivity extends AppCompatActivity {

    public static final String TEST_LOG = "test_log";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_main);

        Log.d(TEST_LOG, "onCreate is start");
    }
    @Override
    protected void onStart(){
        super.onStart();
        Log.d(TEST_LOG, "onResume is start");
    }
    @Override
    protected void onPause(){
        super.onPause();
        Log.d(TEST_LOG, "onPause is start");
    }

    @Override
    protected void onResume(){
        super.onResume();
        Log.d(TEST_LOG, "onResume is start");
    }

}

File: Lab22Toast-student/app/src/main/java/ru/myitschool/lab22toast/MainActivity.kt
package ru.myitschool.lab22toast

import android.os.Bundle
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        Toast.makeText(this, R.string.ncreate, Toast.LENGTH_LONG).show();
    }
    override fun onStart() {
        super.onStart()
        Toast.makeText(this, R.string.nstart, Toast.LENGTH_LONG).show();
    }
     override fun onResume() {
        super.onResume()
        Toast.makeText(this, R.string.nnresume, Toast.LENGTH_LONG).show();
    }
    override fun onPause() {
        super.onPause()
        Toast.makeText(this, R.string.npause, Toast.LENGTH_LONG).show();
    }
    override fun onStop() {
        super.onStop()
        Toast.makeText(this, R.string.nstop, Toast.LENGTH_LONG).show();
    }
    override fun onRestart() {
        super.onRestart()
        Toast.makeText(this, R.string.nrestart, Toast.LENGTH_LONG).show();
    }
    override fun onDestroy() {
        super.onDestroy()
        Toast.makeText(this, R.string.ndestroy, Toast.LENGTH_LONG).show();
    }
}